ws-nextjs-app/utilities/pageRequests/getPageData.ts

```diff
@@ -0,0 +1,84 @@
+import { BFF_FETCH_ERROR } from '#app/lib/logger.const';
+import getToggles from '#app/lib/utilities/getToggles';
+import { FetchError } from '#app/models/types/fetch';
+import PageDataParams from '#app/models/types/pageDataParams';
+import constructPageFetchUrl, {
+  UrlConstructParams,
+} from '#app/routes/utils/constructPageFetchUrl';
+import sendCustomMetric from '#server/utilities/customMetrics';
+import { NON_200_RESPONSE } from '#server/utilities/customMetrics/metrics.const';
+import { PageTypes } from '#app/models/types/global';
+import fetchPageData from '#app/routes/utils/fetchPageData';
+import getAgent from '../undiciAgent';
+
+type LoggerType = {
+  error: (id: string, params: { [key: string]: string | number }) => void;
+};
+
+const getPageData = async (
+  { id, service, rendererEnv, resolvedUrl }: PageDataParams,
+  constructUrlParams: UrlConstructParams,
+  logger: LoggerType,
+  pageType: PageTypes,
+) => {
+  const pathname = `${id}${rendererEnv ? `?renderer_env=${rendererEnv}` : ''}`;
+  const livePageUrl = constructPageFetchUrl(constructUrlParams);
```

Comment:
Remove references to `live`?
```suggestion
  const pageUrl = constructPageFetchUrl(constructUrlParams);
```